app/helpers/helper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
+{
+    public static function index_number($page, $per_page, $index)
+    {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Closing brace must be on a line by itself